apps/settings/serializers/auth/ldap.py

```diff
@@ -85,7 +85,7 @@ class LDAPSettingSerializer(serializers.Serializer):
     )
     AUTH_LDAP_CACHE_TIMEOUT = serializers.IntegerField(
         min_value=0, max_value=3600 * 24 * 30 * 12,
-        default=3600 * 24 * 30,
+        default=0,
         required=False, label=_('User DN cache timeout (s)'),
         help_text=_(
             'Caching the User DN obtained during user login authentication can effectively '
```

Comment:
There is no specific request to look into changes between versions of LDAPSettingSerializer from 2021 to 2024. I would need more information to assess whether there are any differences or potential issues worth mentioning.